src/app/static/src/app/actions.ts

```diff
@@ -51,6 +54,179 @@ export const actions: ActionTree<RootState, RootState> = {
+                            colsminus: "cases_averted_error_minus"
+                        }
+                    },
+                    {
+                        id: "IRS",
+                        name: "Only IRS",
+                        type: "scatter",
+                        marker: {color: "purple", size: 10},
+                        error_x: {
+                            type: "data",
+                            width: 0,
+                            cols: "cases_averted_error_plus",
+                            colsminus: "cases_averted_error_minus"
+                        }
+                    },
+                    {
+                        id: "ITN-IRS",
+                        name: "Add IRS to Pyrethoid ITN",
+                        type: "scatter",
+                        marker: {color: "darkred", size: 10},
+                        error_x: {
+                            type: "data",
+                            width: 0,
+                            cols: "cases_averted_error_plus",
+                            colsminus: "cases_averted_error_minus"
+                        }
+                    },
+                    {
+                        id: "PBO-IRS",
+                        name: "Add IRS to Pyrethoid-PBO ITN",
+                        type: "scatter",
+                        marker: {color: "orange", size: 10},
+                        error_x: {
+                            type: "data",
+                            width: 0,
+                            cols: "cases_averted_error_plus",
+                            colsminus: "cases_averted_error_minus"
+                        }
+                    }
+                ],
+                layout: {
+                    title: "Strategy costs over 3 years",
+                    xaxis: {
+                        title: 'Cases averted per 1,000 people across 3-years',
+                        showline: true,
+                        range: [-10, 800],
+                        tickvals: [0, 100, 200, 300, 400, 500, 600, 700],
+                        autorange: false,
+                        zeroline: false
+                    },
+                    yaxis: {
+                        title: 'Total costs ($10,000 USD)',
+                        showline: true,
+                        range: [-2, 30],
+                        tickvals: [0, 10 , 20],
+                        autorange: false,
+                        zeroline: false
+                    }
+                }
+            });
+    },
+
+    async [RootAction.FetchCostGraphData](context) {
+        //TODO: Fetch from mintr via backend
+        context.commit(RootMutation.AddCostGraphData,
+            [
+                {
+                    intervention: "none",
+                    cost: 0,
+                    cases_averted: 0,
+                    cases_averted_error_minus: 0,
+                    cases_averted_error_plus: 0,
+                    efficacy: 0,
```

Comment:
It would seem to make sense to put cases averted and efficacy values into the same dataset since these graphs differ only on these outcome values, sharing cost values for interventions.